Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -65,13 +75,19 @@ public class RocksDBStore : BaseStore
         /// option.</param>
         /// <param name="maxLogFileSize">The number to configure <c>max_log_file_size</c>
         /// RocksDB option.</param>
+        /// <param name="blockAtEachEpoch">The number of block to store in DB
+        /// for each epoch. 10000 by default.</param>
+        /// <param name="txAtEachEpoch">The number of Transaction to store in DB
+        /// for each epoch. 10000 by default.</param>
         public RocksDBStore(
             string path,
             int blockCacheSize = 512,
             int txCacheSize = 1024,
             ulong? maxTotalWalSize = null,
             ulong? keepLogFileNum = null,
-            ulong? maxLogFileSize = null
+            ulong? maxLogFileSize = null,
+            int txAtEachEpoch = 86400,
```

Comment:
How about using `TimeSpan` instead of `int` for epoch?

Reply:
We cannot use `TimeSpan` here, the constructor needs the default parameter as a compile-time constant. Cannot code like this.

```suggestion
            TimeSpan txAtEachEpoch = TimeSpan.FromSeconds(86400),
```

There is a way to use `TimeSpan` as a default parameter is `TimeSpan? parameter = null`. IMO it is not good at readable.

Reply:
I changed this parameter name to `blockEpochUnitSeconds` and `txEpochUnitSeconds`, mention by @dahlia. I think it is a means to represent `second`. How about this?

Reply:
🆗  fair enough.